packages/drivers/odsp-driver/src/contracts.ts

```diff
@@ -207,3 +207,16 @@ export interface IVersionedValueWithEpoch {
 }
 
 export const persistedCacheValueVersion = 3;
+
+export interface IGetOpsResponse {
+    nonce: string;
+    code: number;
+    messages?: api.ISequencedDocumentMessage[];
```

Comment:
Can add `retryAfter?: number;` to this too

Reply:
Can you please provide some data on when it would be provided and what is expected from client in such case? I.e. can PUSH decide to throttle? Why?

Reply:
We currently will never set this. But I can imagine a scenario where the server/redis is overloaded and the server would want to slow down these calls from the client. So it would be nice if the client could support this at some point. I would expect the client to throttle their calls if it failed and returned a `retryAfter`